project_stock/models/project_task.py

```diff
@@ -74,6 +74,9 @@ class ProjectTask(models.Model):
     )
     stock_analytic_tag_ids = fields.Many2many(
         comodel_name="account.analytic.tag",
+        relation="account_analytic_tag_project_task_stock_rel",
```

Comment:
This will require migration scripts?

Reply:
Yes, it requires renaming the table in migration script (already done).